BSB_lan_defs.h

```diff
@@ -8879,6 +8882,8 @@ PROGMEM_LATE const cmd_t cmdtbl2[]={
 {0x053D3063,  CAT_DIAG_ERZEUGER,    VT_PRESSURE_1000, 8327,  STR8327,  0,                    NULL,         FL_RONLY,     DEV_178_ALL}, // Wasserdruck
 {0x053D3063,  CAT_DIAG_ERZEUGER,    VT_PRESSURE_1000, 8327,  STR8327,  0,                    NULL,         FL_RONLY,     DEV_195_002}, // Wasserdruck
 {0x053D3063,  CAT_DIAG_ERZEUGER,    VT_PRESSURE_WORD, 8327,  STR8327,  0,                    NULL,         FL_RONLY,     DEV_203_ALL}, // Wasserdruck
+// Bax Luna Platinum+: 0x053D3063 return incorrect value. Bug in Baxi?
```

Comment:
Can you post a telegram for this parameter together with the value that is displayed on your heater correctly?

Reply:
My Baxi return "---"
RGT1->HEIZ QUR      053D3063
DC 86 00 0B 06 3D 05 30 63 D2 27
HEIZ->RGT1 ANS      053D3063 01 00 00
DC 80 06 0E 07 05 3D 30 63 01 00 00 F9 0F

But correct value is "1.1 bar" (returned by 0x053D19F0)

Reply:
Ah, ok, but why was this command if then added for your device family and variant? Or did you just figure out that this parameter returned a wrong value? And why didn't you add the ...9F0 command if for your device then?

Reply:
>Or did you just figure out that this parameter returned a wrong value?
Yes, correct.
0x053D19F0 was added somebody earlier and work correct for DEV_123_ALL. So this line just for info.

Reply:
Ah, ok, thanks!